test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,28 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    private void assertHasNull(Object... objects) {assertTrue(Utils.isAnyNull(objects));}
+
+    private void assertNoNull(Object... objects) {assertFalse(Utils.isAnyNull(objects));}
```

Comment:
Not necessary to abstract the `assertTrue(Utils.isAnyNull(...))` as separate method, because asserting the result of the method itself is fine.

Although existing test `assertAreUnique` does that, but that is to prevent `Arrays.asList(...)` from flying everywhere.